runtime/api/Robot.py

```diff
@@ -12,49 +12,494 @@
 >>> from api import Robot
 '''
 import memcache
-
 memcache_port = 12357
 mc = memcache.Client(['127.0.0.1:%d' % memcache_port]) # connect to memcache
 
 motor = {}
 
+def _lookup(name): #Returns actual UID given name
+    return name
+
 def get_motor(name):
     """Returns the current power value for a motor.
 
     :param name: A string that identifies the motor
     :returns: A value between -100 and 100, which is the power level of that motor.
```

Comment:
Currently, joysticks use a -1.0 to 1.0 range, and analog devices use a 0.0 to 1.0 range. The motors should be standardized to match.

I don't think using whole numbers (integers) is any more intuitive than using floats. Moreover, using its quantizes the range (so it can only take on a fixed number of values). However, some devices are capable of returning more fine-grained distinctions in value than 1/100 of their maximum. So I think using floats is the way to go, and capping out at 1.0 is only natural.